packages/types/src/interfaces/tracer.ts

```diff
@@ -45,6 +45,12 @@ export interface Tracer {
    */
   setError(error: Error): NodeSpan | undefined
 
+  /**
+   * Sends an error in a newly created Span that will be closed after
+   * the given error is added to it.
+   */
+  sendError(error: Error): void
```

Comment:
The sendError function should have a second argument, a function that has a span as a parameter, to set additional metadata.

Described here: https://github.com/appsignal/integration-guide/blob/main/features/error-tracking.md#adding-metadata
Example of how it works in the Ruby gem in the docs [send_error Adding metadata section](https://docs.appsignal.com/ruby/instrumentation/exception-handling.html#appsignal-send_error-adding-metadata)

Reply:
Thank you for pointing that out! I added a function with a `NodeSpan` as an argument to add metadata to it.